src/components/MoneyRequestConfirmationList.js

```diff
@@ -241,6 +241,12 @@ function MoneyRequestConfirmationList(props) {
     // A flag for showing the tags field
     const shouldShowTags = props.isPolicyExpenseChat && (props.iouTag || OptionsListUtils.hasEnabledOptions(_.values(policyTagList)));
 
+    // A flag for showing tax rate
+    const shouldShowTaxRate = true;
+
+    // A flag for showing tax rate
+    const shouldShowTaxAmount = true;
+
```

Comment:
We just need a single flag that controls both tax rate and tax amount.
we can name it as shouldShowTax. If true, we'll show both tax rate and tax amount. If false, we'll display neither of them.

For debug, we can keep it true. But it won't be always true.
it can be


`const shouldShowTax = props.isPolicyExpenseChat && props.policy.isTaxTrackingEnabled;`